app/src/main/java/cm/aptoide/pt/install/Install.java

```diff
@@ -97,7 +97,7 @@ public int getVersionCode() {
 
     final Install that = (Install) o;
 
-    if (!versionName.equals(that.versionName)) return false;
+    if (versionName != null && !versionName.equals(that.versionName)) return false;
```

Comment:
I discussed with him and we agreed on putting a generic name on the download factory name likeI had before. The reason why we agreed on doing it on the download level and not the install is because if we need to compare for some reason the download with the install they would have different information, and because the install is being created by using the download information